README.md

```diff
@@ -52,6 +52,10 @@ go install github.com/Boeing/config-file-validator/cmd/validator
 The config-file-validator is built as a statically linked binary which can be downloaded and executed on your target system. Binaries are available for Linux, Windows, and MacOS. Navigate to the [releases](https://github.com/Boeing/config-file-validator/releases) page to download the latest version. Once the binary has been downloaded it needs to be installed by moving the downloaded file to a location on your operating system's PATH.
 
 ## Using
+
+> [!IMPORTANT]
```

Comment:
I feel like the `About` section is more appropriate for this type of clarification. Also, I'm not sure this needs an `[!IMPORTANT]` highlight as this was intedented to just be a clarification - `[!NOTE]` is more appropriate.

Reply:
Sure, let me fix it